src/main/java/se/sundsvall/smloader/service/mapper/CaseMapper.java

```diff
@@ -0,0 +1,34 @@
+package se.sundsvall.smloader.service.mapper;
+
+import se.sundsvall.smloader.integration.db.model.CaseEntity;
+import se.sundsvall.smloader.integration.db.model.CaseMapping;
+import se.sundsvall.smloader.integration.db.model.DeliveryStatus;
+
+import java.time.OffsetDateTime;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static se.sundsvall.smloader.integration.util.XPathUtil.evaluateXPath;
+
+public class CaseMapper {
+
+	private CaseMapper() {}
+
+	public static CaseEntity toCaseEntity(final String id, final byte[] xml) {
+		final var xmlContent = new String(xml, UTF_8);
+		final var result = evaluateXPath(xml, "/FlowInstance/Header/Flow/FamilyID");
+		final var familyId = result.eachText().stream().map(String::trim).findFirst().orElse(null);
+
+		return CaseEntity.create()
+				.withId(id)
+				.withFamilyId(familyId)
+				.withOpenECase(xmlContent)
+				.withDeliveryStatus(DeliveryStatus.PENDING);
+	}
```

Comment:
The id that is used in this method is the flowInstanceId, it is only a basic integer and might have conflicts between different open-e instances. 

We cannot use flowInstanceId as primary key, we will have conflicts. 

We should probably have a UUID primary key, a String flowInstanceId and a String instance. Then create a  uniqueConstraint between flowInstanceId and instance.